app/commands/github/issue/open_for_doc_sync_failure.rb

```diff
@@ -0,0 +1,37 @@
+module Github
+  class Issue
+    class OpenForDocSyncFailure
+      include Mandate
+
+      initialize_with :document, :section, :exception, :git_sha
+
+      def call
+        Github::Issue::Open.(repo, title, body)
+      end
+
+      private
+      def repo
+        Document::REPO_NAME
+      end
+
+      def title
+        "🤖 Document sync error for commit #{git_sha[0..5]}"
```

Comment:
I used "Document sync error" instead of "Sync error", as otherwise we'd have the same title for the track syncing issues and we would miss out on one of the two issues if they occur for the same commit.